src/paths.ts

```diff
@@ -17,8 +19,23 @@ export function convertDebuggerPathToClient(fileUri: string|url.Url, localSource
     if (serverIsWindows) {
         serverPath = serverPath.substr(1);
     }
+    if (typeof pathMapping !== 'undefined') {
+        for (let mappedServerSource of Object.keys(pathMapping) ) {
```

Comment:
mappedServerSource (or mappedServerPath) is modified by the slash normalization below